content/docs/accessibility.md

```diff
@@ -225,32 +224,29 @@ class Parent extends React.Component {
   }
 }
 
-// Now you can set focus when required.
+// Այժմ դուք կարող եք focus-ը տեղադրել երբ անհրաժեշտ է։
 this.inputElement.current.focus();
 ```
 
-When using a HOC to extend components, it is recommended to [forward the ref](/docs/forwarding-refs.html) to the wrapped component using the `forwardRef` function of React. If a third party HOC
-does not implement ref forwarding, the above pattern can still be used as a fallback.
+Երբ կոմպոնենտները ընդլայնելու համար HOC ես օգտագործում, խորհուրդ է տրվում ուղարկել հղումը ներդրված կոմպոնենտին օգտագործելով React֊ի `forwardRef` ֆունկցիան։ Եթե միջնորդ HOC֊ը չի կիրառում հղման ուղարկում, ապա վերը նշված ձևը կարող է օգտագործվել որպես fallback(պահեստային) տարբերակ։
```

Comment:
«խորհուրդ է տրվում ուղարկել հղումը ներդրված կոմպոնենտին» -> էս կոնտեքստում չի հասկացվում, թե «հղումը» ով ա։ Բացի դրանից բնորինակում մի հատ էլ link կա.

it is recommended to [forward the ref](/docs/forwarding-refs.html)

խորհուրդ է տրվում [ուղարկել ref-ը](/docs/forwarding-refs.html) ներդրված կոմպոնենտին